arrow-optics/src/test/kotlin/arrow/optics/OptionalTest.kt

```diff
@@ -22,6 +25,25 @@ class OptionalTest : UnitSpec() {
     init {
 
         testLaws(
```

Comment:
These law test have to be separated into different `testLaws` calls or only one set will run. Will be fixed it in #611.

Also fixes in #611 will fix false negatives in laws.